examples/example_win32_directx12/main.cpp

```diff
@@ -69,13 +69,15 @@ int main(int, char**)
         return 1;
     }
 
+    // Setup Dear ImGui context
+    IMGUI_CHECKVERSION();
+    ImGui::CreateContext();
+
     // Show the window
     ::ShowWindow(hwnd, SW_SHOWDEFAULT);
     ::UpdateWindow(hwnd);
 
-    // Setup Dear ImGui context
-    IMGUI_CHECKVERSION();
-    ImGui::CreateContext();
+    
     ImGuiIO& io = ImGui::GetIO(); (void)io;
```

Comment:
Why moving those CreateContext lines?

Reply:
Hi,

It's because ShowWindow function will call into wmproc, which in turn needs context for most of the messages (WM_SIZE in particular). Thus I moved context creation to before ShowWindow.

Regards,